lastools-el7/fpm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+version=170414
```

Comment:
I don't think changing the version is a good idea, the version currently used, is what you get when you run add the **--version** option to any of the commands. Changing it is a bad idea, who cares if it represents a year or not, more important, to me at least, is that the versions match, changing it, even on the file name is not a good idea.